src/js/components/sharedComponents/autocomplete/Autocomplete.jsx

```diff
@@ -286,7 +286,7 @@ const Autocomplete = (props) => {
                 autocompleteInputRef.current.style.fontWeight = "600";
             }
         }
-    }, [props?.type]);
+    }, [props.selectedItemsDisplayNames, props.type]);
 
```

Comment:
Should we still include the ? for props?.type? Just wondering if we won't run into errors if it doesn't have a props.type.

Reply:
Good eye, because this was one of those automated changes done by our linter, so I did look into it. But after looking at it I think this change is good because the props listed are actually used in the useEffect, and I can't see any problems caused by the new prop being there, or by the lack of the '?' after props.